src/components/BottomBarCourse.vue

```diff
@@ -303,4 +418,72 @@ export default {
     height: 0;
   }
 
+@media only screen and (max-width: 976px) {
+  .bottombarcourse {
+    left: 25.5rem;
+    width: calc(100vw - 25.5rem);
+
+    &-wrapper {
+      display: flex;
+      flex-direction: column;
+      height: 16.25rem;
+      overflow: auto;
+      padding: 0.5rem;
+    }
+
+    &-bar {
+      &-info{
+        &-noOverflow {
+          width: 100%;
+          height: auto;
+          float: none;
+          overflow: none !important;
```

Comment:
!important is usually bad practice. There is some good information here if you want to read some more about why and other options: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity . See if you can fix it here and everywhere else it is in this code in a few minutes, and if you can't, make sure it does not break anything and make an item 😄

Reply:
Removed !important from the file!